src/main/java/com/github/dockerjava/api/model/HostConfig.java

```diff
@@ -75,13 +75,16 @@
     @JsonProperty("CpuShares")
     private int cpuShares = 0;
 
+    @JsonProperty("PidMode")
+    private String pidMode;
+
     public HostConfig() {
     }
 
     public HostConfig(Bind[] binds, Link[] links, LxcConf[] lxcConf, LogConfig logConfig, Ports portBindings,
```

Comment:
Nope. Not needed. It's already removed in 'issue-246' branch. So no need to remove it here.